era5_cloud/create_artifact.jl

```diff
@@ -0,0 +1,86 @@
+using Downloads
+using NCDatasets
+
+using ClimaArtifactsHelper
+
+FILE_URL = "https://caltech.box.com/shared/static/929dvfp42shtxwze00vfkk2i2clz79ty.nc"
```

Comment:
Would it be possible to have an automated download link from some official source?

Reply:
That would be more convenient. I downloaded the data here by selecting a bunch of things: https://cds.climate.copernicus.eu/datasets/reanalysis-era5-single-levels-monthly-means?tab=download. They provide an API but it requires a personal access token.

Reply:
I think it would be helpful if we could set it up. We will download a lot of data from era5 for e.g. calibration.

Reply:
We can do this. We can add a token to ClimaCommon enviroments.